### DISCUSSION TITLE
Quickdraw for own dead PR

### DISCUSSION THREAD

**Discussion by sturmenta** [community/community#27234] [Other Feature Feedback, Questions, & Ideas]
Creating a [pull request](https://github.com/alfredosalzillo/supabase-swr/pull/5) and closing it without merge should not count towards having the achievement

<img width="371" alt="Screen Shot 2022-07-26 at 19 42 02" src="https://user-images.githubusercontent.com/30802967/181124702-aa50c8c5-2827-4aca-9a71-7fc034377484.png">

[<img width="956" alt="Screen Shot 2022-07-26 at 19 42 23" src="https://user-images.githubusercontent.com/30802967/181124709-fae299b1-cbfb-4a81-81f6-66ebf3e59635.png">](https://github.com/alfredosalzillo/supabase-swr/pull/5)

related https://github.com/community/community/discussions/20485

**Comment by radar-blip** [comment]
It would seem that way. But it happens.
